_data-prepper/pipelines/pipelines-configuration-options.md

```diff
@@ -25,6 +25,6 @@ Option | Required | Type        | Description
 :--- | :--- |:------------| :---
 routes | No | String list | A list of routes for which this sink applies. If not provided, this sink receives all events. See [conditional routing]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/pipelines#conditional-routing) for more information.
 tags_target_key | No | String   | When specified, includes event tags in the output of the provided key.
-include_keys | No | String list | When specified, provides the keys in this list in the data sent to the sink.
-exclude_keys | No | String list | When specified, excludes the keys given from the data sent to the sink.
+include_keys | No | String list | When specified, provides the keys in this list in the data sent to the sink. Note that some codecs or sinks may disallow this feature. See the documentation for your sink or codec.
```

Comment:
For both lines, should we call out which codecs and sinks in the description?

Reply:
I was thinking we should not because that may change over time.